### ISSUE TITLE
Add support for Observable Plot?

### ISSUE THREAD

**Issue by ArthurAndrews** [ramnathv/htmlwidgets#484]
I don't know if this is possible, but it would be really neat to have a tighter integration between R and Observable Plot as an htmlwidget.  

It's currently possible to pass data to Observable in Quarto, and this is really powerful, but an htmlwidget would allow users to use Observable Plot from R code alone.

**Comment by timelyportfolio** [comment]
Agreed @juba started https://github.com/juba/obsplot but has appears to have shifted focus to maintaining the [pyobsplot](pyobsplot).  Perhaps we should revive his efforts.

**Comment by ArthurAndrews** [comment]
I just took a look at the `obsplot` package.  Seems very ambitious! in terms of R function interfaces.

I can only speak for myself, but I would be happy with a very plain R `list()` input to `Plot` that's up-to-date with Observable and on CRAN.  

The ```ojs chunks in Quarto are nice, but they limit its use to notebook chunks - not apps, html table cells, etc.  And actually the ```ojs chunks don't yet render in RStudio, so the current workflow is not that nice.  

`htmlwidgets` is great tool and Observable Plot seems like a worthy addition :)

**Comment by juba** [comment]
I can confirm that `obsplot` is not developed anymore and I don't think I will get back to it. That said I'm not sure I would make the same design choices if I had to start over now.

**Comment by timelyportfolio** [comment]
In a very raw and most basic form, here is a quick little attempt at an htmlwidget wrapper [oplot](https://github.com/timelyportfolio/oplot).  The design of Observable Plot does not translate well into htmlwidgets, since Plot uses functions rather than a `json` spec.  I am going to try to think through how we might better support in R without needing an endless stream of `htmlwidgets::JS` or just an inelegant JavaScript `eval`.
